omegaconf/base.py

```diff
@@ -336,7 +336,7 @@ def _select_impl(
         )
         return root, last_key, value
 
-    def _resolve_simple_interpolation(
+    def resolve_simple_interpolation(
```

Comment:
Let's keep it private. I don't mind the library using it's own private methods.

Reply:
Done in ba4d4fe